bitnami/dataplatform-bp1/README.md

```diff
@@ -357,21 +356,44 @@ $ helm install my-release -f values.yaml bitnami/dataplatform-bp1
 
 In the default deployment, the helm chart deploys the data platform with [Metrics Emitter](https://hub.docker.com/r/bitnami/dataplatform-emitter) and [Prometheus Exporter](https://hub.docker.com/r/bitnami/dataplatform-exporter) which emit the health metrics of the data platform which can be integrated with your observability solution.
 
-In case you need to deploy the data platform with [Tanzu Observability](https://docs.wavefront.com/kubernetes.html) Framework for all the applications (Kafka/Spark/Solr) in the data platform, you can specify the 'enabled' parameter using the `--set <component>.metrics.enabled=true` argument to `helm install`. For Solr, the parameter is `solr.exporter.enabled=true` For Example,
+- To deploy the data platform with Tanzu Observability Framework with the Wavefront Collector with enabled annotation based discovery feature for all the applications (Kafka/Spark/Elasticsearch/Logstash) in the data platform, make sure that auto discovery `wavefront.collector.discovery.enabled=true` is enabled, It should be enabled by default and specify the 'enabled' parameter using the `--set <component>.metrics.enabled=true` argument to helm install. For Example,
```

Comment:
This paragraph is duplicated with the line `375`

Reply:
Line number 359 explains about how to deploy wavefront with annotation based discovery enabled and line number 375 explains how to deploy wavefront without the annotation based discovery enabled. Two paragraphs are different.

Reply:
ah, sorry, I didn't notice that